sql/moz-fx-data-shared-prod/mozilla_vpn_derived/subscription_events_v1/query.sql

```diff
@@ -1,3 +1,10 @@
+WITH json_arrays AS (
+  SELECT
+    *,
+    TO_JSON_STRING(promotion_codes) AS json_promotion_codes
```

Comment:
dry run is failing because this field hasn't been deployed yet. I need to deploy the schema update and make sure dry run passes before merging this.